JavaScript1/Week2/homework.md

```diff
@@ -6,200 +6,157 @@
 
-14. We saw that we can pass functions as arguments to other functions. Your task is to write a function that takes another function as an argument and runs it. 
+---
 
-```js
-    function foo(func) {
-        // What to do here? 
-    }
-    
-    function bar() {
-        console.log('Hello, I am bar!');
-    }
-    
-    foo(bar);
-```
+8. Today is Sunday and you ask your friend in how many days you are meeting. If the friend says
+I will see you in 9 days. The output should be Tuesday. You should get the today's day from the system.
+So typical console output is:
 
+``Today is: Sunday``    
+``How many days to meet : 9``  
+``We are meeting on : Tuesday``  
 
-15. Write some code to test two arrays for equality using `==` and `===`. Test the following:
-    
-```js
-var x = [1, 2, 3];
-var y = [1, 2, 3];
-var z = y;
-```
+Hint: use remainder operator
 
-What do you think will happen with `x == y`, `x === y` and `z == y` and `z == x`? Prove it!
-    
-> Don't cheat! Seriously - try it first.
-    
+## Step 3: Smart-ease - Goes Global!
+Following the massive success of Smart-ease's first couple of products things have been quite hectic at Smart-ease's shared office space. The phone is constantly ringing from journalists wanting to interview the genius behind the success and the greatest developers want to work for you. Wired wrote an article with the headline "Smart-ease as good as Smarties"
 
-Check out this [Fiddle](http://jsfiddle.net/jimschubert/85M4z/). You need to open your browser’s Developer Tools to see the console output. Press the Run button in the upper right corner to run the code.
+BUT people are asking: What will be the next product that truly defines Smart-ease as the startup of the century?
 
-More insights from this [Stack Overflow question](http://stackoverflow.com/questions/22395357/how-to-compare-two-arrays-are-equal-using-javascript).
+As always Smart-ease starts by helping people fix their problems. So lets venture out into the real world and see what we find:
 
+### NOnoN0nOYes (Note taking app)
+> You are sitting at a meeting. A person is presenting some interesting thought that you want to write down. You check your bag, but realise that you forgot to bring your notepad. Hmm you check the internet for an online solution. But you need to signup for them all, too bad... Maybe this could be a worthwhile problem so solve. Lets create a simple and easy to use notepad 📝
 
-16. Take a look at the following code: 
+#### Save a note
+The first thing we will create is the functionality to save a note:
+Create a variable called notes and assign it to an empty array.
+Create a function called addNote. The addNote has two parameters: note and noteId. The note parameter is a string. and the noteId is a number.
```

Comment:
should the variables, functions etc be highlighted as code? Just a thought for even easier reading.